modules/main.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable import/extensions */
```

Comment:
- [ ] I don't think this linter disabling is necessary. I removed it and it returned no error for me.